src/query.ts

```diff
@@ -70,39 +86,38 @@ async function run(): Promise<void> {
     chdir(workDir);
 
     try {
-      let dbZip: string;
-      if (repo.downloadUrl) {
-        // 1a. Use the provided signed URL to download the database
-        console.log("Getting database");
-        dbZip = await download(repo.downloadUrl, `${repo.id}.zip`);
-      } else {
-        // 1b. Use the GitHub API to download the database using token
-        console.log("Getting database");
-        dbZip = await downloadDatabase(repo.id, repo.nwo, language, repo.pat);
+      const dbZip = await getDatabase(repo, language);
+
+      if (liveResults) {
+        await setVariantAnalysisRepoInProgress(variantAnalysisId, repo.id);
```

Comment:
Shall we do this before we download the db? (L89)

Reply:
Makes sense, thanks!